packages/eslint-plugin/src/rules/no-floating-promises.ts

```diff
@@ -88,10 +89,21 @@ export default util.createRule<Options, MessageId>({
               suggest: [
                 {
                   messageId: 'floatingFixVoid',
-                  fix(fixer): TSESLint.RuleFix {
-                    let code = sourceCode.getText(node);
```

Comment:
🤔 I'm surprised the build isn't failing - but `sourceCode` is indeed no longer used. Could you remove the import please? Then we can merge!

Reply:
Thank you!
I had overlooked that.

Reply:
I don't know what caused the build to fail either, but it was cured when I merged the main branch.